File: pylox/Token.py

class Token:

    def __init__(self,type,lexeme,literal,line) -> None:
        self.type=type
        self.lexeme=lexeme
        self.literal=literal
        self.line=line
    
    def __str__(self):
        return str(self.type)+" "+str(self.lexeme)+" "+str(self.literal)

File: phlox/Token.php
<?php
namespace Phlox;


class Token {

    private $type;
    private $lexeme;
    private $literal;
    private $line;
     
    function __construct($type, $lexeme, $literal, $line)
    {
        $this->type = $type;
        $this->lexeme = $lexeme;
        $this->literal = $literal;
        $this->line = $line;
    }

    public function __toString():string
    {
        return strval($this->type . " " . $this->lexeme . " ".$this->literal);
    }
    
}

?>